### ISSUE TITLE
embed: ship embed.js for deployment by user

### ISSUE THREAD

**Issue by foliovision** [foliovision/freedom-video-player#249]
Original issue link: [#291 embed: ship embed.js for deployment by user](https://github.com/flowplayer/flowplayer/issues/291)

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/168338?v=4' /> [phloxic](https://github.com/phloxic) commented on Mar 15, 2013

See my comment here: https://github.com/flowplayer/flowplayer/issues/30#issuecomment-14954066
Asked for and required by many customers who's policy does not allow using scripts or content not under their control - for them embedding currently is not an option.

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/168338?v=4' /> [phloxic](https://github.com/phloxic) commented on Mar 15, 2013

@bbbo has already played with that, maybe he can comment when he's back.

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/1395672?v=4' /> [bbbo](https://github.com/bbbo) commented on Apr 3, 2013

- make embed.js a plugin (eg separte file) which can be loaded from releases.flowplayer.org _if_ embedding is not turned off via config, or alternatively from another source if it is defined via flowplayer.config.embed-source: or some such option.
- make embed.js load flowplayer and jquery from either CDN (default) or alternative source if configured via data / flowplayer.config attributes

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/168338?v=4' /> [phloxic](https://github.com/phloxic) commented on Apr 3, 2013

Basically a config clone should be sent to embed.js (the location of which should be configurable too) and embed.js should process that config.

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/265438?v=4' /> [nnarhinen](https://github.com/nnarhinen) commented on Apr 22, 2013

When #358 is merged, embed script location can be set by flowplayer.conf.embed.script configuration variable
